homeassistant/components/google_assistant/trait.py

```diff
@@ -2765,11 +2764,17 @@ def query_attributes(self):
         if device_class is None or data is None:
             return {}
 
-        sensor_data = {"name": data[0], "rawValue": self.state.state}
+        try:
+            value = float(self.state.state)
```

Comment:
This is somewhat surprising, does it mean this has not worked since air quality sensors were added by https://github.com/home-assistant/core/pull/80579 ?

Reply:
It looks like that. The values of my esphome air quality sensors were posted as strings, that gave an unknown result. After converting to float this started working.